### DISCUSSION TITLE
Full Screen option for Remote Desktop Viewer with MeshRouter

### DISCUSSION THREAD

**Discussion by techmonpiano** [Ylianst/MeshCentral#3630] [General]
Good day,

Is it possible to enable "Full Screen" for Remote Desktop Viewer that comes with MeshRouter?

Thanks for your time!

**Comment by techmonpiano** [comment]
Also, for the native "Remote Desktop" portion of MeshRouter, It defaults to 60% quality  Quality setting and frame rate of "Medium."
If I change those settings, they do not "Save" / "stay" if I disconnect and reconnect to a computer.
Is it possible to keep those settings?
Thanks!

**Reply by JSuenram** [reply]
Sounds like a good enhancement request to keep last quality and frame rate settings saved.

**Comment by Ylianst** [comment]
Two good feature requests for sure! I did not realize the quality and frame rate did not save. These should auto-save.

**Reply by techmonpiano** [reply]
Thanks Yllian!  Should I create an "Issue" for this here in Github?  or ?

**Comment by jaswant090** [comment]
please how can i here the sound of the remote system

**Reply by si458** [reply]
I don't believe this is an option sorry 

Is only remote support/control
